go/teams/service_helper.go

```diff
@@ -433,6 +433,7 @@ func reAddMemberAfterResetInner(ctx context.Context, g *libkb.GlobalContext, tea
 	if err != nil {
 		return err
 	}
+	_ = g.Pegboard.TrackUPAK(libkb.NewMetaContext(ctx, g), upak.Current)
```

Comment:
> Also need a way to send down a TrackUV from the front-end when the user clicks "let them back in"
This does that. Frontend calls `ReAddMemberAfterReset` when "let them back in" is tapped. May want to split RPCs or change the name in the future depending on how different the UI is.